code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -2155,9 +2155,12 @@ datum
 						M.bodytemperature += rand(5,20) * mult
 				M.stuttering += rand(0,2)
 				M.bodytemperature += rand(0,3) * mult
-				if(prob(10))
+				if (holder.get_reagent_amount(src.id) >= 3)
+					if(prob(10))
+						M.emote(pick("cough"))
+						M.setStatusMin("stunned", 1 SECOND * mult)
+				if(prob(25))
 					M.emote(pick("cough"))
```

Comment:
this can cough twice, maybe make the stun dependent on coughing and adjust inner prob to match?
```suggestion
				if(prob(25))
					M.emote(pick("cough"))
					if (holder.get_reagent_amount(src.id) >= 3 && prob(40)
						M.setStatusMin("stunned", 1 SECOND * mult)
				
```

Reply:
i wanted to have it so at the lowest leve you still cough so it still has some outward effect, but i will remove it from the stunned one too

Reply:
this makes it where you roll a chance to cough no matter what, but only check to stun if it's over 3 units - i think that is the same as you intended?